src/app/shared/object-list/my-dspace-result-list-element/item-list-preview/themed-item-list-preview.component.html

```diff
@@ -0,0 +1,16 @@
+<div>
+    <div class="row" *ngIf="collection">
+        <div class="col-12">
+            <div class="col-12">
+                <i>
+                    <small>{{"collection.listelement.badge" | translate}}:</small>
+                </i>
+                &nbsp;
+                <a [href]="uiBaseUrl+'/collections/'+collection.uuid">
+                    {{collection.metadata["dc.title"][0]["value"]}}
```

Comment:
I think you can just use `collection.name` here, as that's what is shown on the Collection homepage. We already parse out the title appropriately into a "name" field.